### PR TITLE
Websocket receiver multithreading

### PR DESCRIPTION
**Pull request by LeaveMyYard**
Process relay receiver websocket events in a threadpool. Refactor related code.

### CODE REVIEW

**Review by arikalon1 — COMMENTED** [review]
nice work
left few minor comments

File: src/robusta/integrations/receiver.py [hunk 1]
```diff
@@ -44,6 +47,20 @@ class ValidationResponse(BaseModel):
     error_msg: Optional[str] = None
 
 
+class SlackActionRequest(BaseModel):
+    value: ExternalActionRequest
+
+    @validator("value", pre=True, always=True)
+    def validate_value(cls, v: Union[str, dict]) -> dict:
```
[the remark this answers is not in the document]
Comment (LeaveMyYard) [reply src/robusta/integrations/receiver.py:54]: Despite if always coming as a string in our case, not processing the dict will break Liskov Substitution Principle
So processing both should be better for potential future use

File: src/robusta/integrations/receiver.py [hunk 2]
```diff
@@ -44,6 +47,20 @@ class ValidationResponse(BaseModel):
...
+        # Slack might send the value as a stringified json, so we need to parse it before validation
```
[the remark this answers is not in the document]
Comment (LeaveMyYard) [reply src/robusta/integrations/receiver.py:55]: Will fix it

File: src/robusta/integrations/receiver.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (arikalon1) [reply src/robusta/integrations/receiver.py:54]: I don't think that's required
it's just confusing 
if it's just a string, lets just use a string

**Review by arikalon1 — APPROVED** [review]

**Pull request merged**